docs-source/content/userguide/managing-domains/model-in-image/reusing-rcu.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Reusing an RCU database"
+date = 2020-03-11T16:45:16-05:00
+weight = 70
+pre = "<b> </b>"
+description = "Reusing an RCU database between deployments of a Model in Image JRF domain."
++++
+
+#### Reusing an RCU database between JRF domain deployments
+
+{{% notice info %}} This section only applies for a JRF domain type. Skip it if your domain type is `WLS` or `Restricted JRF`.
+{{% /notice %}}
+
+When you deploy a JRF domain for the first time, the domain will add itself to its RCU database tables, and also create a 'wallet' file in the domain's home directory that enables access to the domain's data in the RCU database. This wallet is encrypted using an OPSS key password that you supply to the domain using a secret.
+
+If it is important to reuse or share the same database and data between deployments of your domain, then it is also important locate and preserve its OPSS wallet password and wallet file. An OPSS wallet password and wallet file allows a JRF deployment to access a FMW infrastructure database that has already been initialized and used before.
```

Comment:
you cant share the OPSS data between domains - not sure what this is talking about

Reply:
FMW Infrastructure has a capital I

Reply:
@markxnelson As per our conversation in slack, we need to zoom about this.  I don't have the background to make updates here.  Please please ping me when you're available to zoom.